components/store/StoreCard.js

```diff
@@ -55,89 +64,108 @@ export default function StoreCard({
     });
   };
 
+  const storeOpenStatus = computeStoreOpen(store.storeHours);
+
   return (
-    <StoreCardContainer includeMargins>
-      <SpaceBetweenRowContainer>
-        <RowContainer>
-          {seeProduct ? (
-            <Subhead
-              style={{
-                marginTop: 2,
-                maxWidth: getMaxWidth(
-                  Dimensions.get('window').width,
-                  snapOrEbtAccepted,
-                  seeProduct
-                ),
-              }}
-              numberOfLines={1}
-              ellipsizeMode="tail">
-              {storeName}
-            </Subhead>
-          ) : (
-            <Title
-              style={{
-                maxWidth: getMaxWidth(
-                  Dimensions.get('window').width,
-                  snapOrEbtAccepted,
-                  seeProduct
-                ),
-              }}
-              numberOfLines={1}
-              ellipsizeMode="tail">
-              {storeName}
-            </Title>
-          )}
-          {snapOrEbtAccepted && (
-            <EBTStatusBar>
-              <FontAwesome5 name="check" size={10} color={Colors.darkerGreen} />
-              <Caption color={Colors.darkerGreen}> EBT</Caption>
-            </EBTStatusBar>
-          )}
-        </RowContainer>
-        {seeProduct && <StoreProductButton callBack={callBack} />}
-      </SpaceBetweenRowContainer>
-      {seeDistance && (
-        <Caption style={{ marginBottom: 4 }} color={Colors.secondaryText}>
-          {`${distance} miles away`}
-        </Caption>
-      )}
+    <TouchableHighlight
```

Comment:
Ok sorry this totally my bad this was a bad suggestion. This turns black on clicking 😅 TouchableOpacity would work just fine instead since it's on StoreCard and not transparent to the whole mapscreen now 😂